Aquifer/Auxiliary.swift

```diff
@@ -46,22 +39,54 @@ associativity left
 precedence 180
 }
 
+/// Returns a pipe that acts like `+++` from Control.Arrow.
+///
+/// `.Left` values fed to the pipe's upstream input will appear downstream as `.Left`s that have
+/// been operated on by the first pipe.  `.Right` values fed to the pipe's upstream input will 
+/// appear downstream as `.Right`s that have been operated on by the second pipe.
 public func +++ <A, B, C, D, R>(p: Proxy<(), A, (), B, R>, q: Proxy<(), C, (), D, R>) -> Proxy<(), Either<A, C>, (), Either<B, D>, R> {
     return left(p) >-> right(q)
 }
 
+/// Returns a pipe that applies the given function to its upstream input.
 public func mapInput<UO, UI, DI, DO, NI, FR>(p: Proxy<UO, UI, DI, DO, FR>, _ f: NI -> UI) -> Proxy<UO, NI, DI, DO, FR> {
     return { request($0).fmap(f) } >>| p
 }
 
+/// Returns a pipe that applies the given function to its downstream output.
 public func mapOutput<UO, UI, DI, DO, NO, FR>(p: Proxy<UO, UI, DI, DO, FR>, _ f: DO -> NO) -> Proxy<UO, UI, DI, NO, FR> {
     return p |>> { v in respond(f(v)) }
 }
 
+/// Yields a pipe that produces left-scanned values with the given step function.
 public func scan1i<DT, FR>(stepWith step: (DT, DT) -> DT) -> Proxy<(), DT, (), DT, FR> {
     return scan1(stepWith: step, initializeWith: identity, extractWith: identity)
 }
 
+/// Yields a pipe that produces left-scanned values with the given step function.  The pipe is not
+/// required to have an initial value, but one is expected to be produced by the `initial` function.
 public func scan1<A, UI, DO, FR>(stepWith step: (A, UI) -> A, initializeWith initial: UI -> A, extractWith extractor: A -> DO) -> Proxy<(), UI, (), DO, FR> {
     return await() >>- { scan(stepWith: step, initializeWith: initial($0), extractWith: extractor) }
 }
+
+/// Implementation Details Follow
```

Comment:
Why is this a doc comment.  Genuinely curious, since I can't see a reason for it.